runtime/onert/core/src/compiler/train/pass/LossInsertionPass.h

```diff
@@ -19,7 +19,7 @@
 
 #include "Pass.h"
 
-#include "compiler/train/TrainingInfo.h"
+#include "ir/Index.h"
```

Comment:
`compiler/train/TrainingInfo.h` include `ir/Index.h`. 
 Since `compiler/train/TrainingInfo` deleted, add `ir/Index.h` explicitly. 

+) `ir::Subgraph` in this file defined in `ir/Index.h`.